pkg/storage/couchdb/couchdbstore.go

```diff
@@ -253,20 +253,44 @@ func (c *CouchDBStore) Put(k string, v []byte) error {
 	return nil
 }
 
-// GetAll fetches all the key-value pairs within this store.
-// TODO: #61 Add support for pagination.
-func (c *CouchDBStore) GetAll() (map[string][]byte, error) {
```

Comment:
Just moved `GetAll` down below `Get` since I think it makes more sense for things to be in a similar order as other documentation I see online... usually the basic functions are first, followed by the more sophisticated one.

(similarly, the new `PutAll` is below `Put`)